app/common/constants/urls.ts

```diff
@@ -0,0 +1,5 @@
+export const FORMS = "/forms"
+export const LOGIN = "/forms/login"
+export const NEW_WALLET = "/forms/wallet"
+
+export const MAIN = "/main"
```

Comment:
IMO, this file should not be in common folder but in renderer, given that these urls won't be used by main process

Reply:
I thought they could come handy if for some reason the backend needed to tell the frontend: hey, go to this url, but yeah I think is better to leave it in renderer and solve that problem in  the future if it's really needed :dancer: